### PR TITLE
Bump go-build

### PR DESCRIPTION
**Pull request by doublek**
## Description

Picks up changes that moves to go 1.17.8 to pick up some CVE remediation fixes

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by robbrockbank — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
